crates/sui-move-build/src/lib.rs

```diff
@@ -726,7 +726,7 @@ pub fn check_unpublished_dependencies(unpublished: &BTreeSet<Symbol>) -> Result<
         .map(|name| {
             format!(
                 "Package dependency \"{name}\" does not specify a published address \
-		 (the Move.toml manifest for \"{name}\" does not contain a published-at field).",
+		 (the Move.toml manifest for \"{name}\" does not contain a 'published-at' field, nor is there a 'published-id' in the Move.lock).",
```

Comment:
This error may now originate from lack of a published-id in either the manifest or lock, so we should mention that. Same below.